aws/resource_aws_lex_bot_test.go

```diff
@@ -0,0 +1,241 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/lexmodelbuildingservice"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAwsLexBot(t *testing.T) {
```

Comment:
This test should be split up. Typically, we prefer that the _basic test is a simple test, possibly with an import and/or simple update. The breakdown for slot types is a good guide.

It would definitely be good to also have a test that validates version publishing.